docs/specification.md

```diff
@@ -146,11 +158,19 @@ Some configuration values are specified in runtime and can be changed during the
 They must be provided on origination to construct the contract's initial storage.
 These values are:
 1. `admin : address` is the address that can perform administrative actions.
-2. `governance_token` is the FA2 contract address/token id pair that will be used as the governance token.
-3. `voting_period : nat` specifies how long the voting period lasts in seconds.
-4. `quorum_threshold : quorum_threshold` specifies what fraction of the frozen
+2. `guardian : address` is the address that have permission to call `drop_proposal`.
+    - `address` of guardian cannot be equal to `SOURCE`.
```

Comment:
I am not sure I haven't mixed up meaning of SENDER/SOURCE, but maybe a better way to put this might be something like "`guardian` cannot initiate the transaction that results in call to drop_proposal".

Reply:
Thanks, that definitely sounds better